tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/ITest.cs

```diff
@@ -3,47 +3,113 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+using System.Reflection;
+using Datadog.Trace.DuckTyping;
```

Comment:
worth adding `#nullable enable` to this one?

Reply:
we can do it, but remember this is a ducktype type, so if we enabled pretty much everything should be marked with `?`